client/src/views/client-area/ClientAreaPage.vue

```diff
@@ -107,7 +107,10 @@
                 v-if="currentPage === ClientAreaPages.CustomOrderBillingDetails"
                 :organization="currentOrganization"
               />
-              <custom-order-processing-page v-if="currentPage === ClientAreaPages.CustomOrderProcessing" />
+              <custom-order-processing-page
```

Comment:
This page will only be displayed if the status is at `NothingLinked` right now.

Reply:
Maybe we can let this `v-if` but with the condition below. With ContractEnded, it will be display on the ContractPage later.
```
if (statusResp.data.status !== CustomOrderStatus.ContractEnded) {
   currentPage.value = ClientAreaPages.CustomOrderProcessing;
 }
```